docs/reference/modules/threadpool.asciidoc

```diff
@@ -1,12 +1,11 @@
 [[modules-threadpool]]
-== Thread Pool
+=== Thread pools
 
 A node holds several thread pools in order to improve how threads memory consumption
 are managed within a node. Many of these pools also have queues associated with them,
 which allow pending requests to be held instead
 of discarded.
```

Comment:
```suggestion

A node uses several thread pools to manage memory consumption. 
Queues associated with many of the thread pools enable pending requests 
to be held instead of discarded. 
```